frontend/src/components/UserSearch.vue

```diff
@@ -138,8 +141,31 @@
 
 <script>
 import UsersRepository from "@/api/UsersRepository";
+import ClientsRepository from "@/api/ClientsRepository";
 import organizations from "@/assets/organizations"
 
+// REVIEW - Is this the best approach to define custom objects using Vue?
+//   Should the custom object be defined in a more Vue-compliant way?
+class ClientRole {
+
+  constructor(role, clients) {
+    this.id = role.id;
+    this.name = role.name;
+    this.description = role.description;
+    this.clientId = role.containerId;
+    this.clientName = clients.find(client => this.clientId === client.id).name;
+  }
+
+  toString() {
+    let s = `${this.clientName} : ${this.name}`;
+    if (typeof (this.description) !== 'undefined') {
```

Comment:
I'm not a Javascript expert, but I think you can just use:

```
if (this.description) {
```

It covers null, undefined, and the empty String.